lib/fork.js

```diff
@@ -3,8 +3,27 @@
 const childProcess = require('child_process')
     , childModule  = require.resolve('./child/index')
 
+let checked, available
+function fork (forkModule, workerOptions, worker_threads) {
+
+  // Check whether we need to run using worker_threads.
+  if (worker_threads) {
+
+    if (!checked) {
+      try {
+        require('worker_threads')
+        available = true
+      } catch (e) {
+        available = false
+        console.warn('[WARNING] Worker threads are not available! Make sure you run node 10.x or higher with --experimental-worker!')
```

Comment:
this'll need to change slightly now that it's unflagged in Node 12+. Maybe something like "Node 12.x or higher, or Node 10.x with the --experimental-worker flag."